### PR TITLE
Add murano barclamp

### PR DESCRIPTION
**Pull request by Itxaka**

### CODE REVIEW

File: chef/roles/murano-server.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name "murano-server"
+description "Murano Server Role - Application Catalog for the cloud"
+run_list(
+  "recipe[murano::common]",
+  "recipe[murano::sql]",
+  "recipe[murano::api]",
+  "recipe[murano::engine]"
```
Comment (vuntz) [inline chef/roles/murano-server.rb]: This is not using what we call "role recipes". Can you check out how we do it for other components and do the same? (in short, we include a single "role_murano_server.rb" recipe, and then from there, we include other recipes).

Comment (Itxaka) [inline chef/roles/murano-server.rb]: done

**Comment by Itxaka** [comment]
rebased

File: bin/crowbar_murano [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+#
+# Copyright 2016, SUSE LINUX GmbH
```
Comment (nicolasbock) [inline bin/crowbar_murano]: The year is outdated now :)

File: chef/cookbooks/murano/attributes/default.rb [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# Copyright 2016, SUSE Linux GmbH
```
Comment (nicolasbock) [inline chef/cookbooks/murano/attributes/default.rb]: year

File: chef/cookbooks/murano/definitions/murano_service.rb [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# Copyright 2016, SUSE Linux GmbH
```
Comment (nicolasbock) [inline chef/cookbooks/murano/definitions/murano_service.rb]: year

File: chef/cookbooks/murano/libraries/helpers.rb [hunk 5]
```diff
@@ -0,0 +1,46 @@
+# Copyright 2016 SUSE Linux GmbH
```
Comment (nicolasbock) [inline chef/cookbooks/murano/libraries/helpers.rb]: year

File: chef/cookbooks/murano/metadata.rb [hunk 6]
```diff
@@ -0,0 +1,28 @@
+# Copyright 2016, SUSE Linux GmbH
```
Comment (nicolasbock) [inline chef/cookbooks/murano/metadata.rb]: year

File: chef/cookbooks/murano/recipes/api.rb [hunk 7]
```diff
@@ -0,0 +1,100 @@
+# Copyright 2016 SUSE Linux GmbH
```
Comment (nicolasbock) [inline chef/cookbooks/murano/recipes/api.rb]: year

File: chef/cookbooks/murano/recipes/common.rb [hunk 8]
```diff
@@ -0,0 +1,50 @@
+# Copyright 2016 SUSE Linux GmbH
```
Comment (nicolasbock) [inline chef/cookbooks/murano/recipes/common.rb]: year

File: chef/cookbooks/murano/recipes/engine.rb [hunk 9]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2016 SUSE Linux GmbH
```
Comment (nicolasbock) [inline chef/cookbooks/murano/recipes/engine.rb]: year

File: chef/cookbooks/murano/recipes/ha.rb [hunk 10]
```diff
@@ -0,0 +1,82 @@
+# Copyright 2016, SUSE Linux GmbH
```
Comment (nicolasbock) [inline chef/cookbooks/murano/recipes/ha.rb]: year

File: chef/cookbooks/murano/recipes/role_murano_server.rb [hunk 11]
```diff
@@ -0,0 +1,23 @@
+#
+# Copyright 2016, SUSE LINUX GmbH
```
Comment (nicolasbock) [inline chef/cookbooks/murano/recipes/role_murano_server.rb]: year

File: chef/cookbooks/murano/recipes/sql.rb [hunk 12]
```diff
@@ -0,0 +1,83 @@
+# Copyright 2016 SUSE Linux GmbH
```
Comment (nicolasbock) [inline chef/cookbooks/murano/recipes/sql.rb]: year

File: chef/data_bags/crowbar/template-murano.json [hunk 13]
```diff
@@ -0,0 +1,62 @@
...
+      "config": {
+        "environment": "murano-base-config",
+        "mode": "full",
+        "transitions": false,
+        "transition_list": [
+        ]
+      }
+    }
+  }
+}
```
Comment (nicolasbock) [inline chef/data_bags/crowbar/template-murano.json]: missing newline?

File: chef/data_bags/crowbar/template-murano.schema [hunk 14]
```diff
@@ -0,0 +1,130 @@
...
+                  "sequence": [ { "type": "str" } ]
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
```
Comment (nicolasbock) [inline chef/data_bags/crowbar/template-murano.schema]: missing newline?

File: crowbar_framework/app/controllers/murano_controller.rb [hunk 15]
```diff
@@ -0,0 +1,25 @@
+#
+# Copyright 2016, SUSE LINUX GmbH
```
Comment (nicolasbock) [inline crowbar_framework/app/controllers/murano_controller.rb]: year

File: crowbar_framework/app/helpers/barclamp/murano_helper.rb [hunk 16]
```diff
@@ -0,0 +1,29 @@
+#
+# Copyright 2016, SUSE LINUX Products GmbH
```
Comment (nicolasbock) [inline crowbar_framework/app/helpers/barclamp/murano_helper.rb]: year

File: crowbar_framework/app/models/murano_service.rb [hunk 17]
```diff
@@ -0,0 +1,112 @@
+# Copyright 2016, SUSE LINUX GmbH
```
Comment (nicolasbock) [inline crowbar_framework/app/models/murano_service.rb]: year

File: crowbar_framework/config/locales/murano/en.yml [hunk 18]
```diff
@@ -0,0 +1,41 @@
+#
+# Copyright 2016, SUSE LINUX GmbH
```
Comment (nicolasbock) [inline crowbar_framework/config/locales/murano/en.yml]: year

File: murano.yml [hunk 19]
```diff
@@ -0,0 +1,38 @@
+#
+# Copyright 2016, SUSE LINUX GmbH
```
Comment (nicolasbock) [inline murano.yml]: year

File: murano.yml [hunk 20]
```diff
@@ -0,0 +1,38 @@
...
+    - 'glance'
+  member:
+    - 'openstack'
+
+crowbar:
+  layout: 1
+  order: 140
+  run_order: 140
+  chef_order: 140
+  proposal_schema_version: 3
```
Comment (nicolasbock) [inline murano.yml]: missing newline?

File: chef/cookbooks/murano/recipes/ha.rb [hunk 21]
```diff
@@ -0,0 +1,82 @@
...
+# limitations under the License.
+#
+
+unless node[:murano][:ha][:enabled]
+  Chef::Log.info("HA disabled for Murano")
+  return
+end
+
+network_settings = MuranoHelper.network_settings(node)
+
```
Comment (Itxaka) [inline chef/cookbooks/murano/recipes/ha.rb]: done

**Comment by Itxaka** [comment]
@nicolasbock Fixed year+newlines

**Comment by Itxaka** [comment]
So log murano, it was nice while it lasted :kissing:

**Pull request closed without merging**